app/src/main/res/values/strings.xml

```diff
@@ -99,4 +99,9 @@
     <!-- WaitActivity -->
     <string name="wait_message_slave_verification_method">Waiting for the other device to choose a verification method…</string>
     <string name="wait_message_partner_rhythm_tapping">Waiting for your partner to finish tapping…</string>
+
+    <!-- NFC -->
+    <string name="descr_gif">This should be a gif representing nfc usage.</string>
```

Comment:
I think that "representing the use of NFC" is better. Now it seems that it's about the extent to which NFC is used (like CPU usage, network usage, etc.)